src/elements/Bookmark/Bookmark.scss

```diff
@@ -133,4 +82,68 @@
     &__provider-name {
         flex-grow: 1;
     }
+
+    @mixin vertical-layout() {
+        flex-flow: column;
+
+        .prezly-slate-bookmark__thumbnail {
+            padding-bottom: 50%; // aspect-ratio: 2:1
+            height: 0;
+            position: relative;
+            min-height: initial;
+            min-width: 100%;
+            max-height: initial;
+            background-size: cover;
+        }
+        .prezly-slate-bookmark__thumbnail-image {
+            position: absolute;
+            left: 0;
+            top: 0;
+            width: 100%;
+            height: 100%;
+        }
+        .prezly-slate-bookmark__title {
+            font-size: 22px;
+        }
+        .prezly-slate-bookmark__description {
+            font-size: $font-size-m;
+        }
+        .prezly-slate-bookmark__provider {
+            margin-top: 13px;
+        }
+    }
+
+    @mixin horizontal-layout() {
+        flex-flow: row;
+
+        .prezly-slate-bookmark__thumbnail {
+            min-width: 180px;
+            min-height: 180px;
+            max-width: 180px;
+        }
+        .prezly-slate-bookmark__thumbnail-image {
+            width: 100%;
+            height: 100%;
+        }
+        .prezly-slate-bookmark__title {
+            font-size: $font-size-l;
+        }
+        .prezly-slate-bookmark__description {
+            font-size: $font-size-s;
+        }
+        .prezly-slate-bookmark__provider {
+            margin-top: 16px;
+        }
+    }
+
+    &--vertical {
+        @include vertical-layout();
+    }
+    &--horizontal {
+        @include horizontal-layout();
```

Comment:
I would wrap this part in a `@media` query as well, so that there are no overlapping properties between two layouts.

Reply:
Makes sense. Will do. Thanks!